bundles/framework/divmanazer/component/FileInput.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * @class Oskari.userinterface.component.FileInput
+ *
+ */
+Oskari.clazz.define('Oskari.userinterface.component.FileInput', function (localization) {
+    this.loc = localization;
+    this.el = null;
+    this._template = {
+        fileBox: _.template('<div class="oskari-fileinput" style="display:none"> '+
+                    '<form method="post" action="", enctype="multipart/form-data" class="box" id="fileinput">'+
+                        '<div class="box__input">'+
+                            '<input type="file" name="file" id="file" class="box__file" />'+
+                            '<label>  <%= fileupload %> <label for="file" style="cursor: pointer;"> <a> <%= link %> </a> </label> </label> '+
+                        '</div>'+
+                        '<div class="box__uploading"> <%= uploading %>&hellip;</div>'+
+                        '<div class="box__success"><%= success %></div>'+
+                        '<div class="box__error"><%= error %></div>'+
```

Comment:
Are these some message placeholders that are toggled visible/hidden? Maybe use just one with an extra class signifying the severity?